### DISCUSSION TITLE
Internet Outage When Running PureDNS/ShuffleDNS

### DISCUSSION THREAD

**Discussion by cupello** [projectdiscovery/shuffledns#410] [Q&A]
Hi everyone,

I’m facing a problem and could use some advice. I’m running PureDNS or ShuffleDNS (both inside Docker on a local machine) to resolve a list of subdomains. However, whenever I start the process, it not only takes down my machine's internet connection but also knocks out the internet for my entire house. This happens even when using default settings or with low concurrency.

Is this expected behavior? How can I efficiently resolve a long list of subdomains without causing this kind of disruption?

Thanks in advance for your help!

**Comment by GeorginaReeder** [comment]
Hey @cupello ! Thanks for your question. That's definitely unusual, running our tools shouldn't knock out internet connections.

@dwisiswant0 do we know if this is indeed related to our tools, or would there be another cause here?

**Comment by dwisiswant0** [comment]
I'm having trouble understanding why this is happening. It seems to me like this is a very particular scenario or edge case, so it's pretty tough to replicate exactly what's going on.

However, I did notice that there seemed to be a similar issue that occurred previously in Nuclei:
- https://github.com/projectdiscovery/nuclei/issues/4781 / https://github.com/orgs/projectdiscovery/discussions/5595
- https://github.com/projectdiscovery/nuclei/issues/5593

Pinging @dogancanbakir and see if he has any insight or knows why the previous problem was brought up in the first place.

**Comment by dogancanbakir** [comment]
Based on the available information, I speculate this issue is specific to the user. It may be related to the router configuration or the ISP, as pointed out in https://github.com/orgs/projectdiscovery/discussions/5595#discussioncomment-10532919.

**Comment by cupello** [comment]
I agree with @dogancanbakir. As @dwisiswant0 links mentioned, using a VPN resolved the issue. Thanks!
